reference.md

```diff
@@ -472,7 +687,9 @@ wpasupplicant installed if you let the ``networkd`` renderer handle wifi.
      ``mii-monitor-interval`` (scalar)
      :    Specifies the interval for MII monitoring (verifying if an interface
           of the bond has carrier). The default is ``0``; which disables MII
-          monitoring.
+          monitoring. This is equivalent to the MIIMonitorSec= field for the
+          networkd backend. If no time suffix is specified, the value will be
+          interpreted as milliseconds.
 
```

Comment:
Having the default time unit be seconds or milliseconds depending on the item seems confusing to me.